English_transcripts/school/Epic Quest/sharedAssets/C1_globe_tutorial_ENGLISH.txt

```diff
@@ -0,0 +1 @@
+Tap on the globe to explore. Tap on this button to see village like yours. Tap on magnifying glass to see what objects are hidden in the scene. Tap on the objects you find. Can you find them all? Tap on this button to see your country Tanzania. Tap on this button to see your continent Africa. Tap on this button to see world map. Tap on the animal button to see videos of animals from around the world. Tap on this button to see Earth and our moon. Tap this button to see the solar system. Tap on this back button to exit to the globe and return to the village. Now you unlocked and discover the all clues of village. Now you can access the special map section by tapping on the globe in my hut any time you like.
```

Comment:
@Shailin Please review this once. If this is synchronized properly with the actions